collector/index.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'pganalyze Collector'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The pganalyze collector periodically queries your database and sends metrics and metadata (as "snapshots") to
+the pganalyze app. This is the only component that talks directly to your database. For details on setting up
+the collector, see the [collector installation guide](/docs/install/02_installing_the_collector). The collector
+is open-source and the code is available [on GitHub](github.com/pganalyze/collector).
+
+
+# Configuration Settings
+
+The collector can be configured either through a `INI` config file (in a normal install) or environment variables
+(typically for running via its Docker image). Most settings can be configured through either mechanism (if both
+are present, the config file takes precedence). Note that a single collector instance can monitor more than one
+database server (though this is not supported when configuring through environment variables). The `[pganalyze]`
+section describes settings that apply to all servers, and the other sections are named after the database server
+they correspond to, and define settings for just that server. When using environment variables, the setting applies
+to all monitored servers.
+
+After you make changes, the config file can be reloaded while the collector is running with the command
+`pganalyze-collector --reload`. This minimizes monitoring interruptions and simplifies config file updates.
+
+The tables below list configuration settings, their defaults if not set, and their descriptions. If a setting is
+configurable through environment variables, the environment variable name follows the setting in parentheses.
+Environment variables for boolean settings expect `1` for true and `0` for false.
+
+
+## General settings
+
+Common settings for configuring monitoring regardless of platform.
+
+<table>
+  <thead>
+    <tr>
+      <th>setting</th>
+      <th>default</th>
+      <th>description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>api_key (PGA_API_KEY)</td>
+      <td>n/a - required</td>
+      <td>
+        API key to authenticate the collector to the pganalyze app. We will show this
+        when adding a server in the app, and you can review it in your organization's
+        API keys page.
+      </td>
+    </tr>
+    <tr>
+      <td>api_base_url (PGA_API_BASEURL)</td>
+      <td>https://api.pganalyze.com</td>
+      <td>URL to contact the pganalyze app. You typically will not need to change this.</td>
```

Comment:
Base URL for contacting the pganalyze API. You typically do not need to change this, unless you are running [pganalyze Enterprise Edition](/docs/enterprise).